pkg/resources/rdma/mlnxRdma.go

```diff
@@ -0,0 +1,38 @@
+package rdma
+
+import (
+	"github.com/Mellanox/rdmamap"
+	pluginapi "k8s.io/kubernetes/pkg/kubelet/apis/deviceplugin/v1beta1"
+
+	"github.com/intel/sriov-network-device-plugin/pkg/types"
+)
+
+// mlnxRdmaDir path to rdma resources
+const mlnxRdmaDir = "/dev/infiniband"
```

Comment:
remove mlnx prefix from here and this const completely.

Add RdmaDeviceDir = "/dev/infiniband" to https://github.com/Mellanox/rdmamap/blob/master/rdma_map.go and use is here as rdmamap.RdmaDeviceDir

Reply:
@Mmduh-483 addition of RdmaDeviceDir is taken care in the https://github.com/Mellanox/rdmamap/blob/master/rdma_map.go 

Please update and use that directly.

Reply:
updated